gpuarray/include/gpuarray/blas.h

```diff
@@ -1,12 +1,12 @@
-#ifndef GPUARRAY_BLAS_H
-#define GPUARRAY_BLAS_H
+#ifndef LIBGPUARRAY_GPUARRAY_BLAS_H
+#define LIBGPUARRAY_GPUARRAY_BLAS_H
 
-#include <gpuarray/buffer_blas.h>
-#include <gpuarray/array.h>
```

Comment:
The includes inside the .h should really keep the angle brackets since they will be used inside user projects.